$NetBSD: patch-qt_mkspecs_solaris-g++-64_qmake.conf,v 1.1 2014/07/02 21:20:08 fhajny Exp $

Do not mess with XOPEN_SOURCE on SunOS. Fixes GCC builds.
--- qt/mkspecs/solaris-g++-64/qmake.conf.orig	2014-06-07 00:54:20.000000000 +0000
+++ qt/mkspecs/solaris-g++-64/qmake.conf
@@ -36,7 +36,7 @@ QMAKE_LEX		= flex
 QMAKE_LEXFLAGS		=
 QMAKE_YACC		= yacc
 QMAKE_YACCFLAGS		= -d
-QMAKE_CFLAGS		= -m64 -D_XOPEN_SOURCE=500 -D__EXTENSIONS__
+QMAKE_CFLAGS		= -m64 -D__EXTENSIONS__
 QMAKE_CFLAGS_DEPS	= -M
 QMAKE_CFLAGS_WARN_ON	= -Wall -W
 QMAKE_CFLAGS_WARN_OFF	= -w
